packages/react-app/src/components/quest.tsx

```diff
@@ -554,6 +561,20 @@ export default function Quest({
       <WrapperStyled>
         {!isEdit && (
           <>
+            <FirstRowStyled className="pb-0">
+              <HeaderRowStyled>
+                <StateTag state={questData?.state ?? ''} />
+              </HeaderRowStyled>
+              <HeaderRowStyled>
+                <AmountFieldInput
+                  id={`balance-${questData?.bounty?.parsedAmount}`}
```

Comment:
Better say its bounty but can be set to unique quest to prevent id duplicates
```suggestion
                  id="bounty-questData?.address"
```